ush/global_cycle.sh

```diff
@@ -58,6 +58,8 @@
 #     FNMXIC        Input maximum sea ice climatology GRIB file.
 #                   Defaults to ${FIXam}/global_maxice.2x2.grb
 #     FNTSFC        Input SST climatology GRIB file.
+#                   Defaults to ${FIXam}/global_salclm.t1534.3072.1536.nc
+#     FNSALC        Input Salinity climatology netcdf file.
```

Comment:
There is a typo here and in the next line.

Reply:
Fixed and pushed.
Thanks!